internal/constants/regexp.go

```diff
@@ -63,11 +64,19 @@ const (
 	RegExpCorporateLastName  = regExpLen128
 
 	// Identity provider
-	RegExpAlias                     = RegExpRealmName
+	RegExpAlias                     = `^EXTIDP-[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$`
```

Comment:
EXTIDP- should be specific to external idp service. The bridge should not be aware of this specific format.
Currently, identity providers are used only for AGW but it could be any identity provider so there is no reason for the alias to only start with EXTIDP-